tools/build-on-openbsd

```diff
@@ -16,12 +22,45 @@ pkgs="
    py3-setuptools
    py3-yaml
    sudo--
+   wget
 "
-[ -f "$depschecked" ] || pkg_add "${pkgs}" || fail "install packages"
 
-touch $depschecked
+[ -f $depschecked ] || echo "Installing the following packages: $pkgs"; output=$(pkg_add -zI $pkgs 2>&1)
+
+
+if  echo "$output" | grep -q -e "Can't find" -e "Ambiguous"; then
+    echo "Failed to find or install one or more packages"
+    echo "Failed Package(s):"
+    echo "$output"
+    exit 1
+else
+    echo Successfully installed packages
+    touch $depschecked
+
+    python3 setup.py build
+    python3 setup.py install -O1 --distro openbsd --skip-build --init-system sysvinit_openbsd
 
-python3 setup.py build
-python3 setup.py install -O1 --distro openbsd --skip-build
+    echo "Installation completed."
+    RC_LOCAL="/etc/rc.local"
+    RC_LOCAL_CONTENT="
+/usr/lib/cloud-init/ds-identify
+wait
```

Comment:
Wait is for background tasks, right? I would understand this if the rc script was itself calling `/etc/rc.d/cloud-init start` (not sure if that's how rc services get invoked, but I think you know what I mean) directly, since that would put the script in the background, but since cloud-init executes syncronously in the foreground I don't think this should be necessary. At least that's what I would expect from bash / posix shell.